ODataConnectedService/src/Templates/ODataT4CodeGenerator.cs

```diff
@@ -2455,14 +2455,14 @@ public void WriteLine(string textToAppend)
     /// </summary>
     public void Write(string format, params object[] args)
     {
-        this.Write(string.Format(global::System.Globalization.CultureInfo.CurrentCulture, format, args));
+        this.Write(string.Format(CultureInfo.InvariantCulture, format, args));
     }
     /// <summary>
     /// Write formatted text directly into the generated output
     /// </summary>
     public void WriteLine(string format, params object[] args)
     {
-        this.WriteLine(string.Format(global::System.Globalization.CultureInfo.CurrentCulture, format, args));
+        this.WriteLine(string.Format(CultureInfo.InvariantCulture, format, args));
```

Comment:
Maybe this change was not meant for checkin?

Reply:
This change was from a previous commit on the tt file but the generated cs file wasn't committed

Reply:
Then the changes should be part of that other PR, not this one.